FlowCrypt/Functionality/Mail Provider/Message Provider/Gmail+MessageExtension.swift

```diff
@@ -103,7 +103,7 @@ private extension GTLRGmail_Message {
 
     func parseMessageBody() -> MessageBody {
         let html = body(type: .textHtml)
-        let text = body(type: .textPlain) ?? html?.removingHtmlTags() ?? ""
+        let text = (body(type: .textPlain) ?? "").removingHtmlTags() ?? ""
```

Comment:
@sosnovsky, Initially, I suspected that the HTML email was displaying as plain text because NSAttributedString initialization didn't recognize the xmlns attribute. 

However, after thorough evaluation, I discovered that my initial assumption was incorrect. The real issue is that `removingHtmlTags` function wasn't invoked for this particular instance.

What's your opinion on this fix?

Reply:
With this fix we'll probably lose message text if there is no plain text body as it'll be converted to `""`?

Reply:
Sorry, it was wrong fix. just pushed new fix